vendor/laravel/lumen-framework/src/Application.php

```diff
@@ -1199,6 +1199,7 @@ public function dispatch($request = null)
         } else {
             $method = $this->getMethod();
             $pathInfo = $this->getPathInfo();
+			$pathInfo = str_ireplace('polr/public/', '', $pathInfo);
```

Comment:
What's the purpose of this line? Would we be able to place this somewhere else, to avoid changing vendor files? I think these files would get overridden if someone tried to run a `composer` update.